addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
No, the `dialogClass` is added but the XML template can define classes. So if you only have to style elements which are in the `.modal-body`, you can add a class `myClass` on the XML template root and the `.modal-body` will have that class `myClass`, the `modal-body` class and the `dialogClass` specified in JS. The XML template defines the very modal body element and the `modal-body` class is automatically added.